storage/boltdb/client.go

```diff
@@ -67,6 +68,7 @@ func NewShared(path string, buckets ...string) ([]*Client, error) {
 	if err != nil {
 		return nil, Error.Wrap(err)
 	}
+	db.NoSync = true
```

Comment:
There are other things that may use bolt.

Reply:
I mostly added this `db.NoSync` setting to start a discussion about if we should be handling the syncing logic in ourselves.  Personally I think we should avoid this setting and instead keep letting BoltDB handling the syncing.  While this setting does seem to improve performance a little bit, I'm not convinced it is worth the increased risk and complication added to the code base.

Anyhow, about your comment...tt looks like this function `NewShared` is currently only used for the Routing table creation. So if we decide that we want to handle syncing ourselves, then syncing logic needs to be added to the routing table code for bootstrap and SA (as it is for SN below).

If we do choose to keep this `db.NoSync = true` in here, I don't know how we can ensure that future uses of this function handle syncing correctly. I could add comment, but it still seems risky.